gltfTutorial/gltfTutorial_001_Introduction.md

```diff
@@ -0,0 +1,38 @@
+
+## Introduction to glTF
+
+There is an increasing number of applications and services that are based on 3D content. Online shops are offering product configurators with a 3D preview. Museums are digitizing their artifacts with 3D scans, and allow exploring their collection in virtual galleries. City planners are using 3D city models for planning and information visualization. Educators are creating interactive, animated 3D models of the human body. In many cases, these applications are running directly in the web browser, which is possible because all modern browsers support efficient, OpenGL-based rendering with WebGL.
+
+![applications](images/applications.png)

```

Comment:
It would be good to have captions for each figure, or label them as `(a), (b), etc.` so the descriptions can be connected.